code/pref.py

```
@@ -0,0 +1,27 @@
+from tensorflow.keras.models import load_model
+
+# Load data and model
+df = pd.read_pickle("job_embeddings.pkl")
+model = load_model("job_recommender_model.h5")
+
+# User preferences
+user_skills = ["Python", "SQL", "Machine Learning"]
+user_experience = 2
+
+# Encode user input
+user_input = encode_user_input(user_skills, user_experience)
```

Ruff (0.8.2)

Undefined name `encode_user_input`

(F821)